cmd/kube-apiserver/app/server.go

```diff
@@ -208,7 +212,29 @@ func Run(s *options.APIServer) error {
 	if err != nil {
 		glog.Errorf("Failed to create clientset: %v", err)
 	}
-	admissionController := admission.NewFromPlugins(client, admissionControlPluginNames, s.AdmissionControlConfigFile)
+
+	plugins := []admission.Interface{}
+	for _, pluginName := range admissionControlPluginNames {
+		plugin := admission.InitPlugin(pluginName, client, s.AdmissionControlConfigFile)
```

Comment:
It almost seems like we're wanting to hide `InitPlugin`.  I'm not sure why I'd ever want to have a way to construct the plugin that didn't include its initialization, right?  It would just end up being an admission plugin that was missing references it needed.

We couldn't reasonably change this downstream in origin, but up in kube we can tweak this to do what we want.